docs/sql-ref-datetime-pattern.md

```diff
@@ -36,11 +36,8 @@ Spark uses pattern letters in the following table for date and timestamp parsing
 |**M/L**|month-of-year|month|7; 07; Jul; July|
 |**d**|day-of-month|number(3)|28|
 |**Q/q**|quarter-of-year|number/text|3; 03; Q3; 3rd quarter|
-|**Y**|week-based-year|year|1996; 96|
-|**w**|week-of-week-based-year|number(2)|27|
-|**W**|week-of-month|number(1)|4|
 |**E**|day-of-week|text|Tue; Tuesday|
-|**u**|localized day-of-week|number/text|2; 02; Tue; Tuesday|
+|**e**|localized day-of-week|number/text|2; 02; Tue; Tuesday|
```

Comment:
shall we forbid `e` as well? It's not supported in 2.4 either.

Reply:
Forgot to update the doc, thanks for reminding